app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -385,6 +395,88 @@ class HomeActivityTest {
     }
   }
 
+  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
+  @Config(qualifiers = "port-mdpi")
+  @Test
+  fun testHomeActivity_phonePort_longProfileName_displaysWelcomeMessageCorrectly() {
+    launch<HomeActivity>(createHomeActivityIntent(longNameInternalProfileId)).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.home_recycler_view)).perform(
```

Comment:
use `scrollToPosition` helper function which is at the bottom of this file.

Reply:
ditto for others

Reply:
Done